packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -417,8 +417,8 @@ export const Checkout = ({
           )}
           data-testid="subscription-create"
         >
-          <Localized id="new-user-step-1">
-            <h2 className="mt-10 text-lg">1. Create a Firefox account</h2>
```

Comment:
`new-user-step-1` can be removed from `packages/fxa-payments-server/src/routes/Checkout/en.ftl` as it is no longer being used anywhere.

Reply:
We have a follow up task for removing the unused strings.